docs-source/content/userguide/istio/istio.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Istio support"
+date: 2019-08-15T13:30:04-04:00
+weight: 1
+---
+
+#### Overview
+
+WebLogic Server Kubernetes Operator version 2.6 and later includes support for Istio 1.4.2 and later.
+This support allows you to run the operator itself, and WebLogic domains managed by
+the operator, with Istio sidecar injection enabled.  It will allow you to use
+Istio gateways and virtual services to access applications deployed in these domains.
+If your applications have suitable tracing code in them, then you will also be able to
+use distributed tracing, such as Jaeger, to trace requests across domains and to
+other components and services that have tracing enabled.
+
+You can learn more about Istio at [Istio](https://istio.io/latest/docs/concepts/what-is-istio/)
+
+#### Limitations
+
+The current support for Istio has these limitations:
+
+* It is tested with Istio 1.4.2 and later (up to 1.5.4), however it is tested with both single and
+  multicluster installations of Istio.
+* You cannot expose any of the default channels, any attempt will result in an error when deploying the domain.  
+* In order to use WLST command, you can define a network access point (NAP) in your WebLogic Domain and expose it as a `NodePort` 
+in your domain resource YAML instead of accessing the channel through the Istio mesh network.  
+
+#### Using the operator with Istio support
+
+{{% notice note %}}
+These instructions assume that you are using a Kubernetes cluster with
+[Istio Installation](https://istio.io/latest/docs/setup/install/) installed and configured already.  The operator will not install
+Istio for you.
+{{% /notice %}}
+
+You can deploy the operator into a namespace which has Istio automatic sidecar
+injection enabled.  Before installing the operator, create the namespace you
+wish to run the operator in, and label it for automatic injection.
+
+```
+$ kubectl create namespace weblogic-operator
+$ kubectl label namespace weblogic-operator istio-injection=enabled
+```
+
+After the namespace is labeled, you can install the operator using the normal
```

Comment:
you can install the operator using the normal method. ->  you can [install the operator]({{< relref "/userguide/managing-operators/installation/_index.md" >}}).